assets/constants.ts

```diff
@@ -563,10 +563,14 @@ export = {
 
   "INLINE_RTE_COMPONENTS": ["link", "math", "skillreview"],
 
+  // If new difficulties are added or the names changed, only the constants
+  // below need to be edited.
+  "SKILL_DIFFICULTY_EASY": "Easy",
+  "SKILL_DIFFICULTY_MEDIUM": "Medium",
+  "SKILL_DIFFICULTY_HARD": "Hard",
+
   "SKILL_DIFFICULTIES": ["Easy", "Medium", "Hard"],
```

Comment:
Perhaps add a backend test to check that SKILL_DIFFICULTIES = [SKILL_DIFFICULTY_EASY, ...] and that the keys of SKILL_DIFFICULTY_LABEL_TO_FLOAT are correct.

Basically, put automated checks in place to make sure that no one can update any of these constants without updating the others too.

Reply:
Done